Documentation/README_Interactable.md

```diff
@@ -109,6 +109,10 @@ You can use Interactable to detect input events other than just OnClick. The Eve
 
 *Example of a hold event*
 
+<img src="../Documentation/Images/Interactable/Event_onClickEffect.png" width="450">
```

Comment:
Since you are adding grip states, it would be nice to also document that in the README.

Reply:
Good call, added some documentation about states that were not previously mentioned.